tests/robot-cases/Group1-Nightly/Common.robot

```diff
@@ -58,7 +58,7 @@ Test Case - Garbage Collection
     Sleep  1
     Wait Until Page Contains  Finished
 
-    ${rc}  ${output}=  Run And Return Rc And Output  curl -u ${HARBOR_ADMIN}:${HARBOR_PASSWORD} -s --insecure -H "Content-Type: application/json" -X GET "https://${ip}/api/system/gc/2/log"
+    ${rc}  ${output}=  Run And Return Rc And Output  curl -u ${HARBOR_ADMIN}:${HARBOR_PASSWORD} -s --insecure -H "Content-Type: application/json" -X GET "https://${ip}/api/system/gc/1/log"
```

Comment:
How about removing the `-s` option and adding `-i` so that we can debug what happens when the API call fails?